frappe/core/doctype/user/user.js

```diff
@@ -27,12 +27,12 @@ frappe.ui.form.on("User", {
 		}
 	},
 
-	role_profile_name: function (frm) {
-		if (frm.doc.role_profile_name) {
+	role_profiles: function (frm) {
```

Comment:
Does this work on tables? :thinking: (not sure honestly)

Reply:
Yes.